src/api/index.ts

```diff
@@ -43,8 +46,20 @@ export default ({
   clientId,
   host,
   retry = {},
-}: SchemaRegistryAPIClientArgs): SchemaRegistryAPIClient =>
-  forge({
+  agent,
+}: SchemaRegistryAPIClientArgs): SchemaRegistryAPIClient => {
+  // TODO: figure out how to only bind agent to host URL
```

Comment:
So the way you'd do this is to pass in the `gatewayConfigs` as a property of the manifest (what gets passed into `forge`). That way we don't change the gateway configs globally, only for this client.

Reply:
https://github.com/tulios/mappersmith/blob/935122577dcc52e1cfc1516d136f67518ec8e952/src/manifest.js#L24

Reply:
I was having issues with the typings of mappersmith (and prettier... forcing a semicolon before parentheses  🤷‍♂️). From looking at the code I believe is this as mappersmith wants as a config.

Reply:
Opened a PR in mappersmith to avoid having to do the gnarly casting to `any`. https://github.com/tulios/mappersmith/pull/211

Let's merge this anyway and then we can hopefully remember to go back and fix this once a new version of mappersmith is out.